test/releases.test.js

```diff
@@ -0,0 +1,68 @@
+        ...baseConfig,
+        'change-title-escapes': '\\'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing <example> (#3) @jetersen\n* `#code_block` (#4) @jetersen\n* Fixes #4 (#5) @Happypig375\n* 2*2 should equal to 4*1 (#6) @jetersen\n* Rename __confgs\\\\confg.yml to __configs\\\\config.yml (#7) @ghost\n* Adds @nullable annotations to the 1*1+2*4 test in `tests.java` (#0) @Happypig375')
+    })
+    it('escapes titles with \\<*_& correctly', () => {
+      const config = {
+        ...baseConfig,
+        'change-title-escapes': '\\<*_&'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing \\<example> (#3) @jetersen\n* `#code_block` (#4) @jetersen\n* Fixes #4 (#5) @Happypig375\n* 2\\*2 should equal to 4\\*1 (#6) @jetersen\n* Rename \\_\\_confgs\\\\confg.yml to \\_\\_configs\\\\config.yml (#7) @ghost\n* Adds @nullable annotations to the 1\\*1+2\\*4 test in `tests.java` (#0) @Happypig375')
+    })
+    it('escapes titles with @s correctly', () => {
+      const config = {
+        ...baseConfig,
+        'change-title-escapes': '@'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing <example> (#3) @jetersen\n* `#code_block` (#4) @jetersen\n* Fixes #4 (#5) @Happypig375\n* 2*2 should equal to 4*1 (#6) @jetersen\n* Rename __confgs\\confg.yml to __configs\\config.yml (#7) @ghost\n* Adds @<!---->nullable annotations to the 1*1+2*4 test in `tests.java` (#0) @Happypig375')
+    })
+    it('escapes titles with @s and #s correctly', () => {
+      const config = {
+        ...baseConfig,
+        'change-title-escapes': '@#'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing <example> (#3) @jetersen\n* `#code_block` (#4) @jetersen\n* Fixes #<!---->4 (#5) @Happypig375\n* 2*2 should equal to 4*1 (#6) @jetersen\n* Rename __confgs\\confg.yml to __configs\\config.yml (#7) @ghost\n* Adds @<!---->nullable annotations to the 1*1+2*4 test in `tests.java` (#0) @Happypig375')
+    })
+    it('escapes titles with \\<@*_&`# correctly', () => {
+      const config = {
+        ...baseConfig,
+        'change-title-escapes': '\\<@*_&`#'
+      }
+      const changelog = generateChangeLog(pullRequests, config)
+      expect(changelog).toEqual('* A1 (#1) @ghost\n* B2 (#2) @ghost\n* Adds missing \\<example> (#3) @jetersen\n* \\`#<!---->code\\_block\\` (#4) @jetersen\n* Fixes #<!---->4 (#5) @Happypig375\n* 2\\*2 should equal to 4\\*1 (#6) @jetersen\n* Rename \\_\\_confgs\\\\confg.yml to \\_\\_configs\\\\config.yml (#7) @ghost\n* Adds @<!---->nullable annotations to the 1\\*1+2\\*4 test in \\`tests.java\\` (#0) @Happypig375')
```

Comment:
instead of hard coding it. you can use `toMatchInlineSnapshot` which will generate the output.